src/main/rx/rx.c

```diff
@@ -285,8 +295,15 @@ void updateRx(uint32_t currentTime)
             needRxSignalBefore = currentTime + DELAY_10_HZ;
             resetPPMDataReceivedState();
         }
-        shouldCheckPulse = rxSignalReceived;
     }
+
+    if (feature(FEATURE_RX_PARALLEL_PWM)) {
```

Comment:
feature(FEATURE_RX_PARALLEL_PWM) is tested twice (https://github.com/cleanflight/cleanflight/pull/1224/files#diff-7cb1b466189980bc90ef231c7828fb58R288)